admin/class-admin-apple-settings.php

```diff
@@ -225,7 +225,7 @@ public function fetch_settings() {
 
 			// Merge settings in the option with defaults.
 			foreach ( $settings->all() as $key => $value ) {
-				$wp_value       = ( empty( $wp_settings[ $key ] ) )
+				$wp_value       = ( ! isset( $wp_settings[ $key ] ) )
```

Comment:
This change was necessary because `0` is a valid value in the new In Article module position setting.